lib/mondrian/olap/connection.rb

```diff
@@ -55,6 +56,7 @@ def connect
           # currently it is assumed that there is just one schema per connection catalog
           @raw_schema = @raw_catalog.getSchemas.first
           @raw_schema_reader = @raw_connection.getMondrianConnection.getSchemaReader
+          @raw_cache_control = @raw_connection.mondrian_connection.get_cache_control(nil)
```

Comment:
One more thing - I prefer to use original Java method names (and not rely on JRuby's magic method name transformations0 as then it is easier to identify which are Ruby and which are Java methods. So in this case:

```ruby
@raw_cache_control = @raw_connection.getMondrianConnection.getCacheControl(nil)
```